test/utils/fakes.ts

```diff
@@ -8,3 +9,13 @@ import { ModAttachr } from "../../src/ModAttachr";
 export function mockModAttachr(settings?: IModAttachrSettings): IModAttachr {
     return new ModAttachr(settings);
 }
+
+/**
+ * Holds keys for custom mod events.
+ */
+export class CustomEventNames extends EventNames {
```

Comment:
I think it would be slightly more standard-conventions-friendly if you called this `FakeEventNames`.